src/cli/cmd/init/mod.rs

```diff
@@ -0,0 +1,347 @@
+        }
+
+        // Other tools
+        for tool in Prompt::multi_select(
+            "Add any of these standard utilities to your environment if you wish",
+            COMMON_TOOLS,
+        )? {
+            let attr = tool.to_lowercase();
+            dev_shell_packages.push(attr);
+        }
+
+        // Nix formatter
+        if Prompt::bool(
+            "Would you like to add the most commonly used Nix formatter (nixpkgs-fmt) to your environment?",
+        )? {
+            dev_shell_packages.push(String::from("nixpkgs-fmt"));
+        }
+
+        let doc_comments =
+            Prompt::bool("Would you like to add helpful doc comments to your flake?")?;
+
+        if Prompt::bool("Would you like to add any environment variables?")? {
+            loop {
+                let name = Prompt::maybe_string("Variable name")?;
+                if let Some(name) = name {
+                    let value = Prompt::maybe_string("Variable value")?;
+                    if let Some(value) = value {
+                        env_vars.insert(name, value);
+                        if !Prompt::bool("Enter another variable?")? {
+                            break;
+                        }
+                    } else {
+                        break;
+                    }
+                } else {
+                    break;
+                }
+            }
+        }
+
+        // If the dev shell will be empty, prompt users to ensure that they still want a flake
+        if dev_shell_packages.is_empty() {
+            if !Prompt::bool("The Nix development environment you've chosen doesn't have any packages in it. Would you still like to create a flake?")? {
+                println!("See you next time!");
+            }
+            return Ok(ExitCode::SUCCESS);
+        }
+
+        dev_shells.insert(
+            String::from("default"),
+            DevShell {
+                packages: dev_shell_packages,
+                env_vars,
+            },
+        );
+
+        let data = TemplateData {
+            description,
+            inputs,
+            systems,
+            dev_shells,
+            overlay_refs: overlay_refs.clone(),
+            overlay_attrs: overlay_attrs.clone(),
+            has_overlays: overlay_refs.len() + overlay_attrs.keys().len() > 0,
+            doc_comments,
+        };
+
+        let flake_string = data.render()?;
+
+        write_file(self.output, flake_string)?;
+
+        if !project.uses_direnv() && Prompt::bool("Are you a direnv user? Select yes if you'd like to add a .envrc file to this project")? {
+            write_file(PathBuf::from(".envrc"), String::from("use flake"))?;
+        } else {
+            println!(
+                "Your flake is ready to go! Run `nix flake show` to see which outputs it provides."
+            );
+        }
+
+        Ok(ExitCode::SUCCESS)
+    }
+}
+
+fn write_file(path: PathBuf, content: String) -> Result<(), FhError> {
```

Comment:
You can just use `std::fs::write(path, contents)` instead of making your own wrapper.